src/data/faq_de.json

```diff
@@ -765,6 +755,16 @@
                 "id": "resolved",
                 "title": "Geklärt",
                 "accordion": [
+                    {
+                        "title": "[Apple/iOS]: Migration auf neues Gerät bzw. Backup/Restore - App funktioniert nicht",
+                        "anchor": "migration_Apple",
+                        "active": true,
+                        "textblock": [
+                            "AKTUELL: Dieses Problem wurde mit Version 1.5.0 behoben. Wenn Sie Ihre Daten über iTunes auf ein neues Gerät übertragen oder ein Backup wiederhergestellt haben, funktioniert die App nicht richtig. Auf den Startseiten landen Sie in einer Endlosschleife und/oder die Risiko-Ermittlung lässt sich nicht einschalten. Der Risikostatus wird nicht angezeigt.",
```

Comment:
Just for consistency with other entries: Can you format AKTUELL like this:

```
<b>AKTUELL</b><br/>
```

and add a separator at the end: `<hr/>`?

Reply:
Like this @heinezen?